gensim/models/wrappers/fasttext.py

```diff
@@ -123,11 +123,8 @@ def __contains__(self, word):
         if word in self.vocab:
             return True
         else:
-            word_ngrams = set(FastText.compute_ngrams(word, self.min_n, self.max_n))
-            if len(word_ngrams & set(self.ngrams.keys())):
-                return True
-            else:
-                return False
+            word_ngrams = FastText.compute_ngrams(word, self.min_n, self.max_n)
+            return any(ng in self.ngrams for ng in word_ngrams)
```

Comment:
It's difficult to say.  As the character ngrams get smaller there is a greater chance of repeats and inputs may not always be english.  Do character ngrams follow a Zipfian distribution?  Even if they do, is the std library loop in the `Set` constructor really that much faster than iterating in pure Python?  Since the list of character ngrams is bound to be relatively small and we're already looping over all of the ngrams, it doesn't seem like making them into a set would get us anything.

When I played around with this before submitting the PR I actually memoized both `__contains__` and `computer_ngrams` in order to get more speed out of it but I didn't think that would be in line with gensim's goal of memory independence.

Also, I think `word_ngrams` should become `character_ngrams`.